userbenchmark/ddp_experiments/README.md

```diff
@@ -54,8 +54,11 @@ Run with the `--csv_out` flag to get this data in csv output for more convenient
 
 Supported options (not-exhaustive):
 * `--job_dir [folder]` to select the shared job folder. It must be accessible by all the child jobs that are spawned.
-* `--distributed [ddp|?]` to select the type of distributed workflow to test
+* `--distributed [ddp_no_static_graph|fsdp]` to select the type of distributed workflow to test
```

Comment:
why the rename here? do you think we'll support both with/without static graph for a while?  Or should we just add static_graph finally and leave it as 'ddp'?

Reply:
this gets passed through to https://github.com/pytorch/benchmark/blob/main/torchbenchmark/util/distributed/core_model/apply_trainer.py where ddp and ddp_no_static_graph are both supported